lib/active_shipping/carrier.rb

```diff
@@ -92,6 +92,16 @@ def find_tracking_info(tracking_number, options = {})
       raise NotImplementedError, "#find_tracking_info is not supported by #{self.class.name}."
     end
 
+    # Get a list of services available for the a specific route
+    #
+    # @param origin_country [String] The country of origin
```

Comment:
I was voluntary being vague here because to follow the steps of `ActiveShipping::Location.country` which can be _either_ a full name or a country code. Should we remove this flexibility here?